src/ray/raylet/scheduling_queue.cc

```diff
@@ -223,6 +223,32 @@ void SchedulingQueue::QueueMethodsWaitingForActorCreation(
   QueueTasks(methods_waiting_for_actor_creation_, tasks);
 }
 
+bool SchedulingQueue::TaskInQueue(const TaskID &task_id) const {
+  if (methods_waiting_for_actor_creation_.TaskInQueue(task_id)) {
+    return true;
+  }
+  if (waiting_tasks_.TaskInQueue(task_id)) {
+    return true;
+  }
+  if (placeable_tasks_.TaskInQueue(task_id)) {
+    return true;
+  }
+  if (ready_tasks_.TaskInQueue(task_id)) {
+    return true;
+  }
+  if (running_tasks_.TaskInQueue(task_id)) {
+    return true;
+  }
+  if (blocked_tasks_.TaskInQueue(task_id)) {
+    return true;
+  }
```

Comment:
just curious, why not just have a single if statement with all of these cases OR'ed together? Performance wise it should be the same, as it will short circuit out as soon as any term of the OR expression returns true.

Reply:
I felt like it was easier to read this way, but will change it to the OR.